compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -353,15 +353,15 @@ struct QuantizeDequantizeWeights final : public luci::CircleNodeMutableVisitor<v
 private:
   // Check if
   // 1. node is const
-  // 2. node was not quantized
+  // 2. node's dtype is float32
   bool is_quantizable(loco::Node *node)
   {
     auto const_node = dynamic_cast<luci::CircleConst *>(node);
     if (not const_node)
       return false;
 
-    // Skip if this is already quantized
-    if (is_quantized(const_node))
+    // Skip if this is not float32
```

Comment:
there are float64 and float16... some models are float16.

Reply:
You're right. But currently we only support quantization of float32. And I think we can remove the float16 tensors in the target model by some pre-processing :)

Reply:
No. float16/64 can enter here. Then `is_quantizable` will return false, and nothing will happen.

Currently, the model will be rejected by `record-minmax` here (https://github.com/Samsung/ONE/pull/9457/files).